{{cookiecutter.project_slug}}/clients/mobile/react-native/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  printWidth: 100,
+  trailingComma: 'all',
+  singleQuote: true,
+  semi: false,
+  tabWidth: 2,
+};
```

Comment:
same here. I wonder how many files are common between both web and mobile work.

Reply:
Right this is the same as web's. 
I wonder whether we could reference an outer file for prettier so we could gather them together somewhere in an higher folder
Having it in the root of the project just makes it work automatically

I haven't tried this out but what do you think about this option?:
https://prettier.io/docs/en/configuration.html#sharing-configurations

We could reference it in all projects and it would be a single source of truth (an npm package that we create)

Reply:
yea, something to refactor later after the dust has settled and we are happy with all these changes.